eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -40,35 +40,84 @@ final class OyRemote implements Objectionary {
     /**
      * The address template.
      */
-    private final String template;
+    private final UrlOy template;
```

Comment:
@mximp Maybe it's better to rename to `OyUrl`?

Reply:
@volodya-lombrozo according to our convention prefix `Oy` means that an object complies with `Objectionary` interface (i.e. implements it). On the other hand name `UrlOy` stresses out that it's an URL but adapted for Objectionary needs.

Reply:
@mximp ok